pkg/alertmanager/alertstore/bucketclient/bucket_client.go

```diff
@@ -110,29 +123,74 @@ func (s *BucketAlertStore) DeleteAlertConfig(ctx context.Context, userID string)
 	return err
 }
 
+// GetFullState implements alertstore.AlertStore.
+func (s *BucketAlertStore) GetFullState(ctx context.Context, userID string) (alertspb.FullStateDesc, error) {
+	bkt := s.getAlertmanagerUserBucket(userID)
+	fs := alertspb.FullStateDesc{}
+
+	err := s.get(ctx, bkt, fullStateName, &fs)
+	if s.amBucket.IsObjNotFoundErr(err) {
+		return fs, alertspb.ErrNotFound
```

Comment:
If we use `alertspb.ErrNotFound` then we have to change its error message because currently it is `alertmanager config not found`.